layouts/shortcodes/feature-gate-list.html

```diff
@@ -0,0 +1,130 @@
+{{/* Kubernetes major version is hard-coded as 1 */}}
+{{/* for simplicity */}}
+{{- $valid_exclusions := (slice "stable" "deprecated" "unstable") -}}
+{{ $state_excludes := slice }}
+{{ if .Get "exclude" }}
+ {{ $state_excludes = (intersect $valid_exclusions (split (.Get "exclude" ) " ") | sort ) }}
+{{ end }}
+<table>
+  <caption>{{ .Inner }}</caption>
+  <colgroup>
+     <col class="feature-name" width="50%">
+     <col span="5" class="feature-multi-details">
+  </colgroup>
+  <thead>
+   <tr>
+     <th>{{ T "feature_heading_name" }}</th>
+     <th>{{ T "feature_heading_default" }}</th>
+     <th>{{ T "feature_heading_stage" }}</th>
+     <th>{{ T "feature_heading_since" }}</th>
+     <th>{{ T "feature_heading_until" }}</th>
+   </tr>
+  </thead>
+  <tbody>
+  {{- range $featureGate := sort $.Site.Data.features.featureList.featureGateList "name" -}}
+    {{- range $versionUnderConsideration := (seq 1 (int (index (split ( site.Params.version ) "." ) 1 ) ) ) -}}
```

Comment:
Version comparison is a complex business. This simplified version break when major version changes.

Reply:
Yes, that's true. For a major version change we will have some rewriting to do generally.